### PR TITLE
chore(dryrun): minor fixes and use default format for analysis_config and analysis_results [PLT-5368]

### PR DESCRIPTION
**Pull request by siddhant-deepsource**
- Use default format for the `analysis_config.json` and `analysis_results.json`.
- Fix the resolution of `CODE_PATH` and `TOOLBOX_PATH` env vars. Set their value as `/code` and `/toolbox` respectively.
- Fix `echo` logs.

### CODE REVIEW

File: command/analyzer/dryrun/source_test.go [hunk 1]
```diff
@@ -72,7 +72,7 @@ func TestPrepareAnalysisConfig(t *testing.T) {
 			t.Errorf("Failed to read the expected analysis config. Error:%s", err)
 		}
 
-		if !bytes.Equal(receivedAnalysisConfig, expectedAnalysisConfig) {
+		if cmp.Equal(receivedAnalysisConfig, expectedAnalysisConfig) {
```
Comment (tushar-deepsource) [inline command/analyzer/dryrun/source_test.go:75]: consider doing `!= 0` so the intent is clear.

```suggestion
		if cmp.Equal(receivedAnalysisConfig, expectedAnalysisConfig) != 0 {
```

Comment (siddhant-deepsource) [reply command/analyzer/dryrun/source_test.go:75]: Doesn't `cmp.Equal` already return a `bool`? 🤔

Comment (siddhant-deepsource) [inline command/analyzer/dryrun/source_test.go:75]:
```suggestion
		if !cmp.Equal(receivedAnalysisConfig, expectedAnalysisConfig) {
```

Comment (siddhant-deepsource) [reply command/analyzer/dryrun/source_test.go]: Nice catch. Thanks! `s/cmp.Equal/!cmp.Equal`

**Review by sourya-deepsource — APPROVED** [review]

**Pull request merged**